en/docs/references/ballerina-expression-syntax.md

```diff
@@ -0,0 +1,421 @@
+# Ballerina Expression Syntax
+
+Ballerina expressions comprise a powerful expression syntax based on the [Ballerina language](https://ballerina.io/) for function invocations, declaring a variable using literals and tuples, typecasting, etc. This document presents you with the common expressions that you can use while developing Choreo applications.
+
+## Supported Literals 
+
+### String
+
+A string is simply a sequence of characters. It is required to use double quotes (`"`)  to mark the boundary between the starting and ending characters.
+
+#### Examples
+
+  ```ballerina
+  "Colombo"
+  ```
+  ```ballerina
+  "12345"
+  ```
+  ```ballerina
+  "Ballerina is a programming language intended for network-distributed applications."
+  ```
+
+### Signed Integer
+
+A signed integer is an integer with the  `+` or `-` character in the front, indicating the sign. You can also define a positive integer or zero using only numerical characters without the sign. Using the signed integer type, you can give positive integers, negative integers, and zero as an input to an expression-supported field.
+
+#### Examples
+
+  ```ballerina
+  1234
+  ```
+  ```ballerina
+  -10
+  ```
+  ```ballerina
+  +2020
+  ```
+
+### Decimal Floating-Point Number
```

Comment:
Do we need to say `Number` here? As for the previous ones we simply use the type name (i.e. `Signed Integer` and `String` : Not `Signed Integer Number`)

Reply:
No I think, when you say DFP, it's generally used together with another word to emphasize if it's the number you are talking about or the operation. But when you say signed integers it is clear to the reader that it refers to the number.  Hence, used the word 'number'  in DFP to give more clarity to the reader.

Reply:
Ack